crates/runtime/src/modelruntime/tract.rs

```diff
@@ -0,0 +1,129 @@
+use super::ModelRuntime;
+use super::Runnable;
+use arrow::array::ArrayRef;
+use arrow::array::Float32Array;
+use arrow::array::Float64Array;
+use arrow::datatypes::DataType;
+use arrow::datatypes::Field;
+use arrow::datatypes::Schema;
+use arrow::record_batch::RecordBatch;
+use snafu::ResultExt;
+use std::sync::Arc;
+
+use tract_core::tract_data::itertools::Itertools;
+use tract_onnx::prelude::*;
+
+pub struct Tract {
+    pub path: String,
+}
+
+type Plan = SimplePlan<TypedFact, Box<dyn TypedOp>, Graph<TypedFact, Box<dyn TypedOp>>>;
+pub struct Model {
+    model: Plan,
+}
+
+impl Model {
+    fn run_inference(
```

Comment:
nit: We should just put this straight into `fn run(&self, input: Vec<RecordBatch>) -> super::Result<RecordBatch> {
`